src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -89,24 +93,34 @@ export class SkyFileAttachmentComponent implements AfterViewInit, AfterContentIn
     return this._disabled;
   }
 
-/**
- * Specifies the maximum size in bytes for valid files.
- */
+  /**
+   * Specifies the maximum size in bytes for valid files.
+   */
   @Input()
   public maxFileSize: number = 500000;
 
-/**
- * Specifies the minimum size in bytes for valid files.
- */
+  /**
+   * Specifies the minimum size in bytes for valid files.
+   */
   @Input()
   public minFileSize: number = 0;
 
+  /**
+   * Specifies a custom validation function. This validation is ran along with all internal
+   * file validation. This function should take a `SkyFileItem` in as a parameter.
```

Comment:
```suggestion
   * file validation. This function takes a `SkyFileItem` interface as a parameter.
```

Reply:
Hmmm - I know that technically that `SkyFileItem` is an interface; however, the function will take in an instance of that interface so I don't love saying that it takes in the interface. How about this?

"This function takes a `SkyFileItem` object as a parameter."

I've made my suggested change to both this and the file drop component but we can tweak it again if needed.

Reply:
Sounds good. Sorry I got it wrong. I was just trying to avoid `should` to avoid sounding uncertain, and it felt like we needed something after `SkyFileItem`. If `object works better than `interface`, that works for me. Thanks!